samples/csharp/common/ModelBuilder.cs

```diff
@@ -16,9 +14,7 @@ public class ModelBuilder<TObservation, TPrediction>
         public IEstimator<ITransformer> TrainingPipeline { get; private set; }
         public ITransformer TrainedModel { get; private set; }
 
-        public ModelBuilder(
-            MLContext mlContext,
-            IEstimator<ITransformer> dataProcessPipeline //, IEstimator<ITransformer> trainer
+        public ModelBuilder(MLContext mlContext,IEstimator<ITransformer> dataProcessPipeline //, IEstimator<ITransformer> trainer
```

Comment:
Remove link to ModelBuilder.cs  from your project. I don't think you are using ModelBuilder.cs at all.
We want to get rid of this helper class so users directly see the ML.NET API instead of a façade class.
In your case, I think you are not using it. Just remove the link from the project.

Reply:
Ok Cesar